<html>
    <head>

        <link rel="stylesheet" href="../assets/content/bioinformatics/achievements.css">
        <script src="../assets/content/bioinformatics/achievements.js" defer></script>

    </head>
    <body>
        <div class = "papyrus-container">

            <div class = "content">

                <h1 class="text-6xl"><b>Notable achievements</b></h1>
                <br>
                <br>

                <div class="slideshow-container">

                    <!-- Full-width slides/quotes -->
                    <div class="mySlides">
                        <p style="color: brown;">Protein analysis was the starting point</p>
                        <p>The first crystallographic applications of stored-program computers were done on
                            EDSAC and the Manchester Mark II in 1952–1953. However, these were used for inorganic
                            structures. The first application of computers to protein crystallography, which some consider
                            the real forerunner of today's bioinformatics, was in fact for the first high-resolution
                            structure, that of myoglobin, in 1958.
                        </p>
                        <br/>
                        <p>In the late 1950s, the first sequence (i.e. amino acid chain arrangement) of a protein, insulin, was published. This major leap settled the debate about the polypeptide chain arrangement of proteins. The Edman degradation method emerged as a simple method that allowed protein sequencing. Coupled with automation (Figure 1 - One of the first automated peptide sequencers, designed by William J. Dreyer.), more than 15 different protein families were sequenced over the following 10 years.</p>
                        <br/>
                        <p>A major issue with Edman sequencing was obtaining large protein sequences. Edman sequencing works through one-by-one cleavage of N-terminal amino acid residues with phenylisothiocyanate. However, the yield of this reaction is never complete. Because of this, a theoretical maximum of 50–60 amino acids can be sequenced in a single Edman reaction. Larger proteins must be cleaved into smaller fragments, which are then separated and individually sequenced.
                            The issue was not sequencing a protein in itself but rather assembling the whole protein sequence from hundreds of small Edman peptide sequences. For large proteins made of several hundreds (if not thousands) of residues, getting back the final sequence was cumbersome. In the early 1960s, one of the first known bioinformatics software was developed to solve this problem.
                        </p>
                    </div>

                    <div class="mySlides">
                      <p style="color: brown;">The success of Human Genome Project</p>
                      <p>In April 2003, sequencing of all three billion nucleotides in the human genome was declared complete. This landmark of modern science brought with it high hopes for the understanding and treatment of human genetic disorders. There is plenty of evidence to suggest that the hopes will become reality—1631 human genetic diseases are now associated with known DNA sequences, compared to the less than 100 that were known at the initiation of the Human Genome Project (HGP) in 1990. </p>

                    </div>

                    <div class="mySlides">
                      <p style="color: brown;">Second-generation sequencing</p>
                      <p>DNA sequencing was democratised with the advent of second-generation sequencing (also called next-generation sequencing or NGS) that started with the ‘454’ pyrosequencing technology. This technology allowed sequencing thousands to millions of DNA molecules in a single machine run, thus raising again the old computational challenge. The gold-standard tool
                        to handle reads from 454 (the high-throughput sequencer) is still today the proprietary tool Newbler, that was maintained by Roche until the phasing out of the 454 in 2016.
                        </p>
                    </div>

                    <div class="mySlides">
                        <p style="color: brown;">The Integration of Artificial Intelligence and Machine Learning</p>
                        <p>Integrating AI and ML in bioinformatics has already led to significant breakthroughs. For example, researchers at the University of California, San Francisco, used machine learning algorithms to analyse genomic data from more than 10,000 tumours. The algorithms were able to identify patterns that could predict which tumours were likely to respond to a particular type of immunotherapy, improve patient outcomes and reduce the need for expensive and time-consuming clinical trials.
                        </p>
                        <br/>
                        <p>Another example of the potential of AI and ML in bioinformatics is the development of deep learning algorithms for predicting protein structures. Proteins are the building blocks of life and play a critical role in many biological processes. However, predicting the structure of a protein from its amino acid sequence is a complex and challenging task. Deep learning algorithms have shown promise in this area, with researchers at the University of Washington developing a deep learning algorithm that can predict the structure of a protein with near-atomic accuracy.</p>
                        <br/>
                        <p>In drug discovery, AI and ML algorithms have already been used to identify new drug candidates, predict the efficacy of drugs, and reduce the risk of adverse side effects. These algorithms can analyse large data sets of drug-target interactions, drug structures, and molecular properties to identify potential drug candidates and help pharmaceutical companies to streamline the drug discovery process, reducing the time and cost of developing new drugs.</p>
                        <br/>
                        <p>Another area where AI and ML are applied in bioinformatics is developing predictive models for disease diagnosis and prognosis. These models can analyse patient data, such as genomic data, medical history, and clinical data, to predict the likelihood of developing a particular disease, the severity of the disease, and the likely response to treatment.</p>
                      </div>

                    <!-- Next/prev buttons -->
                    <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
                    <a class="next" onclick="plusSlides(1)">&#10095;</a>
                  </div>

                  <!-- Dots/bullets/indicators -->
                  <div class="dot-container">
                    <span class="dot" onclick="currentSlide(1)"></span>
                    <span class="dot" onclick="currentSlide(2)"></span>
                    <span class="dot" onclick="currentSlide(3)"></span>
                    <span class="dot" onclick="currentSlide(3)"></span>
                  </div>
            </div>
        </div>
    </body>
</html>

